images/win/Windows2019-Readme.md

```diff
@@ -325,6 +325,34 @@ This version is saved but not installed
 _Location:_ C:\Modules\azurerm_6.7.0\AzureRM\6.7.0\AzureRM.psd1
 
 
+## Az Powershell modules

```

Comment:
We dont need to update the readme as this will be generated dynamically

Reply:
It is done